packages/react-ui-ag/src/Banners/Banner.js

```diff
@@ -0,0 +1,48 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import themed from 'react-themed'
+import { Text } from '@rentpath/react-ui-core'
+
+@themed(['Banner'], {
+  pure: true,
+})
+
+export default class Banner extends PureComponent {
+  static propTypes = {
+    className: PropTypes.string,
+    theme: PropTypes.object,
+    name: PropTypes.node.isRequired,
+    value: PropTypes.any,
+    nodeType: PropTypes.string,
+  }
+
+  static defaultProps = {
+    theme: {},
+    nodeType: 'div',
+  }
+
+  render() {
+    const {
+      theme,
+      name,
+      nodeType: NodeType,
```

Comment:
I don't see this being used anywhere.

Reply:
That is leftover from the tag component, forgot to delete it. I'll do that now